init.js

```diff
@@ -2,9 +2,61 @@
 
 /**
  * Archetype `init` configuration.
+ *
+ * See: https://github.com/FormidableLabs/builder-init/blob/master/README.md#archetype-data
+ * for structuring this configuration file.
  */
-// TODO: REMOVE AND IMPLEMENT PROMPTS
-// https://github.com/FormidableLabs/builder-init/issues/3
-/*eslint-disable no-console*/
-console.log("TODO HERE init.js");
-/*eslint-enable no-console*/
+module.exports = {
+  // Prompts are Inquirer question objects.
+  // https://github.com/SBoudrias/Inquirer.js#question
+  //
+  // `builder-init` accepts:
+  // - an array of question objects (with a `name` property)
+  // - an object of question objects (keyed by `name`)
+  //
+  prompts: {
+    licenseDate: {
+      message: "License date",
+      default: (new Date()).getFullYear().toString()
+    },
+    licenseOrg: {
```

Comment:
@boygirl -- If we're going to spit out a License, then it _should_ have a name. I don't yet have the capability to completely ignore a file and not produce it (say we wanted to get complicated and _not_ produce the license or package.json specification). Please open an issue if you'd like to see this capability down the road.